### ISSUE TITLE
3.12 Installation failure: EACCES linux-x64-11_binding.node

### ISSUE THREAD

**Issue by alexjurkiewicz** [sass/node-sass#1799]
I think the same issue as I reported for 3.11 in #1780. Log from our CI:

```
error 14-Nov-2016 10:53:19  http node-sass install Downloading binary from https://github.com/sass/node-sass/releases/download/v3.12.0/linux-x64-11_binding.node
error 14-Nov-2016 10:53:24  http node-sass install Download complete
error 14-Nov-2016 10:53:24  info node-sass install Binary saved at /home/bamboo/xml-data/build-dir/MACSF-BMACSFW17-JOB1/website/node_modules/node-sass/vendor/linux-x64-11/binding.node
error 14-Nov-2016 10:53:24  info node-sass install Caching binary to /root/.npm/node-sass/3.12.0/linux-x64-11_binding.node
error 14-Nov-2016 10:53:24
error 14-Nov-2016 10:53:24  events.js:72
error 14-Nov-2016 10:53:24          throw er; // Unhandled 'error' event
error 14-Nov-2016 10:53:24                ^
error 14-Nov-2016 10:53:24  Error: EACCES, open '/root/.npm/node-sass/3.12.0/linux-x64-11_binding.node'
error 14-Nov-2016 10:53:27  npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
error 14-Nov-2016 10:53:27  npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.0.15
error 14-Nov-2016 10:53:27  npm WARN service-finder@1.0.0 No repository field.
error 14-Nov-2016 10:53:27  npm ERR! Linux 3.10.0-327.13.1.el7.x86_64
error 14-Nov-2016 10:53:27  npm ERR! argv "node" "/usr/bin/npm" "install"
error 14-Nov-2016 10:53:27  npm ERR! node v0.10.25
error 14-Nov-2016 10:53:27  npm ERR! npm  v3.10.3
error 14-Nov-2016 10:53:27  npm ERR! code ELIFECYCLE
error 14-Nov-2016 10:53:27
error 14-Nov-2016 10:53:27  npm ERR! node-sass@3.12.0 install: `node scripts/install.js`
error 14-Nov-2016 10:53:27  npm ERR! Exit status 8
error 14-Nov-2016 10:53:27  npm ERR!
error 14-Nov-2016 10:53:27  npm ERR! Failed at the node-sass@3.12.0 install script 'node scripts/install.js'.
error 14-Nov-2016 10:53:27  npm ERR! Make sure you have the latest version of node.js and npm installed.
error 14-Nov-2016 10:53:27  npm ERR! If you do, this is most likely a problem with the node-sass package,
error 14-Nov-2016 10:53:27  npm ERR! not with npm itself.
error 14-Nov-2016 10:53:27  npm ERR! Tell the author that this fails on your system:
error 14-Nov-2016 10:53:27  npm ERR!     node scripts/install.js
error 14-Nov-2016 10:53:27  npm ERR! You can get information on how to open an issue for this project with:
error 14-Nov-2016 10:53:27  npm ERR!     npm bugs node-sass
error 14-Nov-2016 10:53:27  npm ERR! Or if that isn't available, you can get their info via:
error 14-Nov-2016 10:53:27  npm ERR!     npm owner ls node-sass
error 14-Nov-2016 10:53:27  npm ERR! There is likely additional logging output above.
error 14-Nov-2016 10:53:28
error 14-Nov-2016 10:53:28  npm ERR! Please include the following file with any support request:
error 14-Nov-2016 10:53:28  npm ERR!     /home/bamboo/xml-data/build-dir/MACSF-BMACSFW17-JOB1/website/npm-debug.log
```

Our environment info is in #1780.

**Comment by xzyfer** [comment]
Can you please try with v3.12.1

**Comment by alexjurkiewicz** [comment]
Sorry, it seems to still fail :(

error	14-Nov-2016 16:20:30	http node-sass install Downloading binary from https://github.com/sass/node-sass/releases/download/v3.12.1/linux-x64-11_binding.node
error	14-Nov-2016 16:20:35	http node-sass install Download complete
error	14-Nov-2016 16:20:35	info node-sass install Binary saved at /home/bamboo/xml-data/build-dir/ACG-BMACW94-JOB1/website/node_modules/node-sass/vendor/linux-x64-11/binding.node
error	14-Nov-2016 16:20:35	info node-sass install Caching binary to /root/.npm/node-sass/3.12.1/linux-x64-11_binding.node
error	14-Nov-2016 16:20:35	
error	14-Nov-2016 16:20:35	/home/bamboo/xml-data/build-dir/ACG-BMACW94-JOB1/website/node_modules/node-sass/scripts/install.js:140
error	14-Nov-2016 16:20:35	          .on('error', function (e) { throw e; });
error	14-Nov-2016 16:20:35	                                            ^
error	14-Nov-2016 16:20:35	Error: EACCES, open '/root/.npm/node-sass/3.12.1/linux-x64-11_binding.node'

**Comment by xzyfer** [comment]
Hmm ok, please try v3.12.2

**Comment by xzyfer** [comment]
Please don't edit comments. It makes it's hard to follow what's happening. I'm confused. Is it failing to install or just logging an error?

**Comment by alexjurkiewicz** [comment]
Sure, I'll avoid editing posts in future.
3.12.2 output:

```build	14-Nov-2016 16:38:00	> node-sass@3.12.2 install /home/bamboo/xml-data/build-dir/ACG-BMACW94-JOB1/website/node_modules/node-sass
build	14-Nov-2016 16:38:00	> node scripts/install.js
build	14-Nov-2016 16:38:00	
error	14-Nov-2016 16:38:00	http node-sass install Downloading binary from https://github.com/sass/node-sass/releases/download/v3.12.2/linux-x64-11_binding.node
error	14-Nov-2016 16:38:07	http node-sass install Download complete
error	14-Nov-2016 16:38:07	info node-sass install Binary saved at /home/bamboo/xml-data/build-dir/ACG-BMACW94-JOB1/website/node_modules/node-sass/vendor/linux-x64-11/binding.node
error	14-Nov-2016 16:38:07	info node-sass install Caching binary to /root/.npm/node-sass/3.12.2/linux-x64-11_binding.node
error	14-Nov-2016 16:38:07	ERR! node-sass install Failed to cache binary: undefined
build	14-Nov-2016 16:38:09	
build	14-Nov-2016 16:38:09	> node-sass@3.12.2 postinstall /home/bamboo/xml-data/build-dir/ACG-BMACW94-JOB1/website/node_modules/node-sass
build	14-Nov-2016 16:38:09	> node scripts/build.js
build	14-Nov-2016 16:38:09	
error	14-Nov-2016 16:38:09	info node-sass build Binary found at /home/bamboo/xml-data/build-dir/ACG-BMACW94-JOB1/website/node_modules/node-sass/vendor/linux-x64-11/binding.node
error	14-Nov-2016 16:38:09	info node-sass build Testing binary
error	14-Nov-2016 16:38:09	info node-sass build Binary is fine
```

13.12.2 is logging an error but seems to work now :)

**Comment by xzyfer** [comment]
Thanks again, I'll clean up that error message soon.
